app_config/app_config.py

```
@@ -0,0 +1,42 @@
+"""
+Look up sensitive configuration items such as credentials and keys
+so they don't need to be hardcoded as artifacts in the code repository.
+This implementation finds it's resources in the DynamoDB table called 'config_provider'
+"""
+
+import boto
+import boto.dynamodb
+import json
+from boto.exception import BotoClientError
+from collections import defaultdict
+
+
+class AppConfig(object):
+    _ENVIRONMENT_DEFAULT_NAME = 'default'
+    _DYNAMO_DB_TABLE = 'app_config'
+
+    def __init__(self, region, environment_name, table_name=_DYNAMO_DB_TABLE):
+
+        self._resources = defaultdict(dict)
+        self._environment = environment_name
+
+        _conn = boto.dynamodb.connect_to_region(region_name=region)
+        self._table = _conn.get_table(table_name)
+
+    def __getitem__(self, resource_name):
+        if not self._resources[resource_name]:
+            self._load_resources_from_dynamo_db(resource_name, self._ENVIRONMENT_DEFAULT_NAME)
+            self._load_resources_from_dynamo_db(resource_name, self._environment)
+
+        return self._resources[resource_name]
+
+    def _load_resources_from_dynamo_db(self, resource_name, environment):
+        try:
+            item = self._table.get_item(hash_key=resource_name, range_key=environment)
+            if item:
+                resource_dict = json.loads(item.get("config"))
+                self._resources[resource_name].update(resource_dict)
+        except BotoClientError as e:
+            print(e.message)
```

Is it ok to just swallow this exception?